templates/prometheus.yml.j2

```diff
@@ -32,21 +32,25 @@ scrape_configs:
   static_configs:
     - targets: []
   metric_relabel_configs:
+      # in: openio.score;OPENIO.account_10_240_0_23_6009
+      # out: account
     - source_labels: [chart, dimension]
-      regex: openio.score;.*\.(.*?)\_.*
-      replacement: ${1}
+      regex: openio\.score;[^.]+\.([^_]+)_.+
       target_label: service
+      # in: openio.score;OPENIO.account_10_240_0_23_6009
+      # out: 10_240_0_23_6009
     - source_labels: [chart, dimension]
-      regex: openio.score;.*\.(.*?)\_(.*)
-      replacement: ${2}
+      regex: openio\.score;[^.]+\.[^_]+_(.+)
       target_label: s_instance
+      # in: openio.byte_avail;OPENIO.10_240_0_23_6001.2579104859421935362
+      # out: 10_240_0_23_6001
     - source_labels: [chart, dimension]
-      regex: openio.*[byte_avail|byte_used].*;.*\.(.*?)\..*
-      replacement: ${1}
+      regex: openio\.byte_(?:avail|free|used);[^.]+\.([^.]+)\..+
```

Comment:
`byte_free` was not part of the regex before, but was still matched by side-effect, so I added it as I was not sure what was the wanted behavior